src/kontrol/__main__.py

```diff
@@ -80,6 +75,7 @@ def main() -> None:
     sys.setrecursionlimit(15000000)
     parser = _create_argument_parser()
     args = parser.parse_args()
+    args = read_toml_args(parser, args, sys.argv[2:]) if hasattr(args, 'config_file') else args
```

Comment:
Would it be possible for `read_toml_args` to return a `dict` of arguments read from the toml file instead of passing them again to argparse `parse_args` again along with the original command line args? So we could do something like
```
    args_dict = vars(parser.parse_args())
    args_dict = args_dict | read_toml_args()
    ...
    execute(args_dict)
```
This would ideally remove the necessity of having to convert the toml args to a command line format.